sql/core/src/main/scala/org/apache/spark/sql/execution/SparkSqlParser.scala

```diff
@@ -367,6 +367,22 @@ class SparkSqlAstBuilder extends AstBuilder {
   }
 
   /**
+   * Create a [[DropTable]] command.
+   */
+  override def visitDropTable(ctx: DropTableContext): LogicalPlan = withOrigin(ctx) {
+    if (ctx.PURGE != null) {
+      logWarning("PURGE option is ignored.")
```

Comment:
Here, I will only verify if the data are deleted at first, assuming the deleted data is moved to trash directory by Hive when `fs.trash.interval` is set to a positive value. 

Will try to support `ifPurge` in a separate PR. Thanks!